presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1693,9 +1705,15 @@ private void checkHoldsLock()
         private final T data;
         private final String user;
         private final String queryId;
+        private final Map<ActionParameter, Boolean> actionParameters;
```

Comment:
We already have action parameters: the `data` field. This flag is specific to creating a table, so it should go in `TableAndMore`.